modules/gapi/test/common/gapi_tests_common.hpp

```diff
@@ -169,6 +185,10 @@ using compare_f = std::function<bool(const cv::Mat &a, const cv::Mat &b)>;
 
 using compare_scalar_f = std::function<bool(const cv::Scalar &a, const cv::Scalar &b)>;
 
+template<typename pointtype>
+using compare_vectorPoints_f = std::function<bool(const std::vector<cv::Point_<pointtype>> &a,
```

Comment:
Does this template usage look good in this place? As optFlow kernel is WIP, I can say that vectors comparison function would be needed.
But this template goes like vector<Point_<T>> while optFlow would require vector<uchar> and vector<double> so the template should be vector<T>. Am I right?

Reply:
comparison of vectors is unified to goodF and optFlow. Some posiible issues as wrong/inexact sorting of two compared vectors are not taken into account